(err) => {
        if (err)
            throw err;
        console.log('log olu≈üturuldu');
    }